frontends/pytorch/e2e_testing/torchscript/quantized_models.py

```diff
@@ -0,0 +1,58 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import torch
+from torch import nn
+
+from torch_mlir.torchscript.e2e_test.framework import TestUtils
+from torch_mlir.torchscript.e2e_test.registry import register_test_case
+from torch_mlir.torchscript.annotations import annotate_args, export
+
+# ==============================================================================
+
+
+class QuantizedMLP(nn.Module):
```

Comment:
I have a bunch of questions on this PR but may have them all answered if I could see the chunk of actual IR we import for this test case. Any chance you could throw that in a gist?

Reply:
This is the [starting IR](https://gist.github.com/silvasean/47f918f18aa4272aad20f680f1bb8f05)
This is what we manage to get it down to [IR](https://gist.github.com/silvasean/f76bccd76e9b193d396cfb2f9a11f54d)